core/chains/evm/gas/rollups/l1_oracle.go

```diff
@@ -63,13 +63,13 @@ func NewL1GasOracle(lggr logger.Logger, ethClient l1OracleClient, chainType chai
 			l1Oracle, err = NewArbitrumL1GasOracle(lggr, ethClient)
 		case toml.DAOracleZKSync:
 			l1Oracle = NewZkSyncL1GasOracle(lggr, ethClient)
-		default:
-			err = fmt.Errorf("unsupported DA oracle type %s. Going forward all chain configs should specify an oracle type", daOracle.OracleType())
 		}
 		if err != nil {
 			return nil, fmt.Errorf("failed to initialize L1 oracle for chaintype %s: %w", chainType, err)
 		}
-		return l1Oracle, nil
+		if l1Oracle != nil {
+			return l1Oracle, nil
+		}
```

Comment:
Could you also add a test for a scenario where daOracle configs are omitted so it falls back to chain type? Notice the Arbitrum and zkSync tests were using the oracle type which is why this probably slipped through the cracks.